test/serving.bash

```diff
@@ -86,8 +86,17 @@ function run_knative_serving_rolling_upgrade_tests {
 
   failed=0
   image_template="registry.svc.ci.openshift.org/openshift/knative-${KNATIVE_SERVING_VERSION}:knative-serving-test-{{.Name}}"
+  PROBE_FRACTION=1.0
+  prev_serving_version=$(oc get knativeserving.operator.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.version}")
 
+  prev_version=$(echo ${prev_serving_version} | awk -F. '{printf "%2d%02d%02d", $1,$2,$3}') # Format serving version e.g. 0.13.2 to 01302
```

Comment:
I'm curious if this is necessary. Could we just compare the strings alphabetically? e.g. `if [[ $prev_version < "0.14.0" ]]`